### ISSUE TITLE
Maybe there is a bug in your code?

### ISSUE THREAD

**Issue by AnaZou** [sergiomsilva/alpr-unconstrained#36]
Hello,sergriomsilva．Thanks to your code,I have read your source code,but i doubt there maybe b bug in license-plate-detection.py, after detect_lp function,when save the detected plate result,the source code is 
```
if len(LlpImgs):
	Ilp = LlpImgs[0]
	Ilp = cv2.cvtColor(Ilp, cv2.COLOR_BGR2GRAY)
	Ilp = cv2.cvtColor(Ilp, cv2.COLOR_GRAY2BGR)

	s = Shape(Llp[0].pts)
	cv2.imwrite('%s/%s_lp.png' % (output_dir,bname),Ilp*255.)
        writeShapes('%s/%s_lp.txt' % (output_dir,bname),[s])`  
```
In fact,the cvtColor maybe loss color info,maybe should like this:

```
if len(LlpImgs):
	# 按照自己修改的，不会改变图像的颜色
	Ilp = LlpImgs[0]
	Ilp=(Ilp*255).astype("uint8")

	s = Shape(Llp[0].pts)
	cv2.imwrite('%s/%s_lp.png' % (output_dir, bname), Ilp)
	writeShapes('%s/%s_lp.txt' % (output_dir,bname),[s])
```

**Comment by sergiomsilva** [comment]
Hi @AnaZou , good point. Actually, we did this on purpose. We wanted to use grayscale images instead of RBG, however, all backbone models have 3 channels in the input layer. So this part of the code is to ensure the grayscale. I know this is not the best solution, but we hypothesized that colour information with such a few amounts of training samples could lead to unwanted overfitting in the unconstrained scenario. Probably is a better idea to train with colour information for a single country or layout.
